### PR TITLE
A bunch of bugs

### PR DESCRIPTION
**Pull request by GhenadieVP**
Several small bug fixes:
- Proper resources refresh on pull to refresh.
- Properly display the txID.
- Properly display the title of the Choose Asset in Asset Transfer.
- Add back TX message sent from Asset Transfer.

### CODE REVIEW

**Comment by kugel3** [comment]
I have a better solution for the id, ResourceBalance shouldn't be ID:ed at all really, it's only in the TransactionReview context that we use that, and that's really only so that we can apply guarantees. I use `Identified<ResourceBalance>` there instead, and remove the id from ResourceBalance. Using the resource address is not enough, since there are situations where you can have lists where the same resource appears multiple times, etc. 

About the other bugs, what are they?

File: RadixWallet/Clients/AccountPortfoliosClient/AccountPortfoliosClient+Live.swift
Comment (GhenadieVP) [inline RadixWallet/Clients/AccountPortfoliosClient/AccountPortfoliosClient+Live.swift:1]: applied the proper forcerRefresh.

File: RadixWallet/Features/AssetTransferFeature/AssetTransfer+Reducer.swift [hunk 1]
```diff
@@ -77,7 +77,7 @@ public struct AssetTransfer: Sendable, FeatureReducer {
 				let manifest = try await createManifest(accounts)
 				Task {
 					_ = try await dappInteractionClient.addWalletInteraction(
-						.transaction(.init(send: .init(transactionManifest: manifest))),
+						.transaction(.init(send: .init(transactionManifest: manifest, message: message))),
```
Comment (GhenadieVP) [inline RadixWallet/Features/AssetTransferFeature/AssetTransfer+Reducer.swift:80]: Message was removed in some of older PRs

File: RadixWallet/Features/AssetTransferFeature/Components/TransferAccountList/TransferAccountList+View.swift [hunk 2]
```diff
@@ -90,11 +90,11 @@ private extension View {
 	private func addAsset(with destinationStore: PresentationStoreOf<TransferAccountList.Destination>) -> some View {
 		sheet(store: destinationStore.scope(state: \.state.addAsset, action: \.addAsset)) { assetsStore in
 			AssetsView.View(store: assetsStore)
+				.navigationTitle(L10n.AssetTransfer.AddAssets.navigationTitle)
```
Comment (GhenadieVP) [inline RadixWallet/Features/AssetTransferFeature/Components/TransferAccountList/TransferAccountList+View.swift:93]: With latest iOS updates the order matters now.

**Review by kugel3 — APPROVED** [review]

**Pull request merged**